### ISSUE TITLE
UI tests are failing on `master`

### ISSUE THREAD

**Issue by willdurand** [mozilla/addons-frontend#7572]
I guess the issue is related to a change in FF nightly, which likely requires a change in FoxPuppet.

**Comment by willdurand** [comment]
This depends on https://github.com/mozilla/FoxPuppet/pull/243 (or some other similar patch, although this one works)
